standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -249,7 +249,7 @@ public static boolean isDbReplIncompatible(Database db) {
   public static boolean isDbBeingFailedOver(Database db) {
     assert (db != null);
     Map<String, String> dbParameters = db.getParameters();
-    return dbParameters != null && ReplConst.TRUE.equalsIgnoreCase(dbParameters.get(ReplConst.REPL_FAILOVER_ENABLED));
+    return dbParameters != null && !StringUtils.isEmpty(dbParameters.get(ReplConst.REPL_FAILOVER_ENABLED));
```

Comment:
It can't be any random string, it has to be a valid Failover_Point (i.e FailoverEndpoint as per the other comment)